app/views/includes/link.scala.html

```diff
@@ -33,9 +33,9 @@
     @if(isButton){
         role="button"
         @if(isSecondary){
-            class="button--secondary @linkClasses.mkString(" ")"
```

Comment:
does this secondary not matter?

Reply:
I'm not sure, what does secondary even do?

Reply:
good shout, I need to add `govuk-button--secondary`